packages/dropdowns/src/Menu/Menu.tsx

```diff
@@ -5,21 +5,47 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import React, { createContext, useRef, useEffect } from 'react';
+import React, { createContext, useRef, useEffect, CSSProperties, HTMLProps } from 'react';
 import PropTypes from 'prop-types';
 import { Popper } from 'react-popper';
+import { Modifiers } from 'popper.js';
 import { withTheme, isRtl } from '@zendeskgarden/react-theming';
 
-import { StyledMenu } from '../styled/index.js';
+import { StyledMenu } from '../styled/index';
 import useDropdownContext from '../utils/useDropdownContext';
-import { getPopperPlacement, getRtlPopperPlacement } from '../utils/garden-placements';
-
-export const MenuContext = createContext();
+import {
+  GARDEN_PLACEMENT,
+  getPopperPlacement,
+  getRtlPopperPlacement
+} from '../utils/garden-placements';
+
+export interface IMenuContext {
+  itemIndexRef: React.MutableRefObject<number>;
```

Comment:
Could you import `MutableRefObject` like you've done for `CSSProperties` and `HTMLProps`?